BuildResidentialHPXML/measure.rb

```diff
@@ -3557,6 +3562,21 @@ def self.set_header(hpxml, args)
     if args[:utility_bill_scenario_names].is_initialized
       bills_scenario_names = args[:utility_bill_scenario_names].get.split(',').map(&:strip)
 
+      if args[:utility_bill_electricity_filepaths].is_initialized
+        bills_electricity_filepaths = args[:utility_bill_electricity_filepaths].get.split(',').map(&:strip)
+        bills_electricity_filepaths.each_with_index do |bills_electricity_filepath, i|
+          begin
+            FilePath.check_path(bills_electricity_filepath,
+                                File.dirname(hpxml_path),
+                                'Tariff File')
+          rescue
+            bills_electricity_filepaths[i] = nil
+          end
```

Comment:
I don't like the idea of simply ignoring any file paths we don't find. Also, we don't do this for any other similar input in this measure (schedules_filepaths, emissions_electricity_values_or_filepaths, etc.), so we should be consistent.

I would propose just passing the bills_electricity_filepath to the HPXML file without checking anything, and let the downstream HPXMLtoOpenStudio measure take care of checking the path. It looks like there are a few code changes related to making the `hpxml_path` available in this method, so that all could be reverted.